Makefile

```diff
@@ -2,73 +2,143 @@ MAKEFLAGS 				+= --warn-undefined-variables
+
+configurations/semsim1/hp-mp2.semsim.scrambled1.tsv: configurations/semsim1/hp-mp.semsim.tsv
+	pheval-utils semsim-scramble --input $< --output $@ --scramble-factor 0.5
+
+#CONVERT SAMPLE
+configurations/semsim1/hp-mp.semsim.scrambled1.sql: configurations/semsim1/hp-mp.semsim.scrambled1.tsv
+	pheval-utils semsim-convert --input $< --output $@ --subject-prefix HP --object-prefix MP
+
+configurations/semsim1/hp-mp2.semsim.scrambled1.sql: configurations/semsim1/hp-mp2.semsim.scrambled1.tsv
+	pheval-utils semsim-convert --input $< --output $@ --subject-prefix HP --object-prefix MP
+
+configurations/exomiser-13.01/default/2209_phenotype/2209_phenotype.h2.db: configurations/exomiser-13.01/default/config.yml configurations/semsim1/hp-mp.semsim.scrambled1.sql configurations/semsim1/hp-mp2.semsim.scrambled1.sql
+	test -d $(EXOMISER_DATA_FOLDER)/hp-mp-semsim.scrambled1 || cp -rf configurations/exomiser-13.01/default/2209_phenotype/ $(EXOMISER_DATA_FOLDER)/hp-mp-semsim.scrambled1
+	java -cp $(H2_JAR) org.h2.tools.RunScript -user sa -url jdbc:h2:/$(EXOMISER_DATA_FOLDER)/hp-mp-semsim.scrambled1/2209_phenotype -script configurations/semsim1/hp-mp.semsim.scrambled1.sql
+
+	test -d $(EXOMISER_DATA_FOLDER)/hp-mp2-semsim.scrambled1 || cp -rf configurations/exomiser-13.01/default/2209_phenotype/ $(EXOMISER_DATA_FOLDER)/hp-mp2-semsim.scrambled1
+	java -cp $(H2_JAR) org.h2.tools.RunScript -user sa -url jdbc:h2:/$(EXOMISER_DATA_FOLDER)/hp-mp2-semsim.scrambled1/2209_phenotype -script configurations/semsim1/hp-mp2.semsim.scrambled1.sql
+
+results/phen2gene/default-corpus1-scrambled1/results.yml: configurations/exomiser-13.01/default/config.yml
+	pheval run --input-dir $(PHEN2GENE_LIB) --testdata-dir $(shell pwd)/corpora/corpus1/scrambled1 --runner phen2genephevalrunner --tmp-dir $(shell pwd)/$(TMP_DATA)/ --output-dir $(shell dirname pwd)/$(shell dirname $@) --config $<
+	touch $@
+
+results/exomiser-13.01/default-corpus1-scrambled1/results.yml: configurations/exomiser-13.01/default/config.yml corpora/corpus1/scrambled1/corpus.yml
+	mkdir -p $(shell dirname $@)
+	pheval run --input-dir $(shell pwd)/configurations/exomiser-13.01/default --testdata-dir $(shell pwd)/corpora/corpus1/scrambled1 --runner exomiserphevalrunner --tmp-dir $(shell pwd)/$(TMP_DATA)/ --output-dir $(shell pwd)/$(shell dirname $@) --config $(shell pwd)/$<
+	touch $@
+
+corpora/corpus1/scrambled1/corpus.yml: $(TEST_DATA)/template_vcf/template_exome_hg19.vcf.gz
+	test -d $(shell dirname $@)/vcf || mkdir -p $(shell dirname $@)/vcf
+	test -d $(shell dirname $@)/phenopackets || mkdir -p $(shell dirname $@)/phenopackets
+	test -L $(shell pwd)/corpora/corpus1/scrambled1/template_exome_hg19.vcf.gz || ln -s $(shell pwd)/$< $(shell dirname $@)/vcf/
+	pheval-utils create-spiked-vcfs --template-vcf-path $(shell dirname $@)/vcf/template_exome_hg19.vcf.gz --phenopacket-dir=$(TEST_DATA)/phenopackets/single --output-dir $(shell dirname $@)/vcf
```

Comment:
Correct spirit but I think we should avoid variables for now (test data) best have this goal depend on the default dir with the original corpus information?